### PR TITLE
Add com.mojang.Minecraft

### PR DESCRIPTION
**Pull request by Kekun**

### CODE REVIEW

**Comment by TingPing** [comment]
For now I believe our policy is only free software though there have been discussions of allowing proprietary ones via extra-data.

**Comment by Kekun** [comment]
That's understandable, and in this specific case it uses extra-data.

File: com.mojang.Minecraft.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          "name": "xrandr",
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "git://anongit.freedesktop.org/xorg/app/xrandr"
+              }
+          ]
+      },
+      {
+          "name": "jre",
```
Comment (Barthalion) [inline com.mojang.Minecraft.json]: I'm sure we can't redistribute this either.

File: com.mojang.Minecraft.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      "--extra-data=Minecraft.jar:495ee255cca5c2280e3f57ef93a7db8eb17a4dee233374a6b74d564eb1ec65af:48112736:48112736:http://s3.amazonaws.com/Minecraft.Download/launcher/Minecraft.jar"
+  ],
+  "modules": [
+      {
+          /* Needed by Minecraft 1.8.2 and up. */
+          "name": "xrandr",
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "git://anongit.freedesktop.org/xorg/app/xrandr"
```
Comment (Barthalion) [inline com.mojang.Minecraft.json]: Should use a latest stable instead.

File: com.mojang.Minecraft.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                  "sha256": "66d8971b240bbd03d114308958fff0fb8068d0589122885d9591727e5472137f",
+                  "dest-filename": "jre.tar.gz"
+              }
+          ]
+      },
+      {
+          "name": "minecraft",
+          "no-autogen": true,
+          "buildsystem": "simple",
+          "build-commands": [],
```
Comment (Barthalion) [inline com.mojang.Minecraft.json]: IMHO `build-commands` fits better what you're doing in `post-install` later.

File: com.mojang.Minecraft.json (hunk 1 above)
Comment (Kekun) [inline com.mojang.Minecraft.json]: Maybe OpenJDK would fit better also.

**Comment by nedrichards** [comment]
We've 'clarified' the flathub stance on extra-data now (we have Spotify etc.) so this shouldn't block minecraft any more.

**Comment by Barthalion** [comment]
@Kekun, any updates on this one?

**Comment by Kekun** [comment]
@Barthalion unfortunately no, i spent some time trying to bootstrap openjdk in the flatpak-builder manifest but with no success, and moved on…

Having a Java runtime or module would help greatly here.

**Comment by Alexander-Wilms** [comment]
There's now a JDK extension in Flathub.

**Pull request closed without merging**